streams/src/test/java/org/apache/kafka/streams/TopologyTest.java

```diff
@@ -423,10 +423,8 @@ public void process(final Record<Object, Object> record) { }
         }
     }
 
-    @Deprecated // testing old PAPI
```

Comment:
Test is already updated to use the new PAPI -- removing this outdated annotation